Sources/Shared/API/Models/Action.swift

```diff
@@ -22,6 +22,7 @@ public final class Action: Object, ImmutableMappable, UpdatableModel {
     @objc public dynamic var Scene: RLMScene?
     @objc public dynamic var isServerControlled: Bool = false
     @objc public dynamic var serverIdentifier: String = ""
+    @objc public dynamic var CarPlayAvailable: Bool = true
```

Comment:
maybe more like "shouldShowInCarPlay"? you don't need to follow the super old "capitalize the first letter" rule for new things, but "available" sounds like it may be unavailable for some reason more than just being hidden

Reply:
Updated